app/views/components/blockEditor.html

```diff
@@ -41,12 +41,23 @@
     <!-- Tab panes -->
     <tabset>
       <tab heading="Options">
-        <div class="form-group">
+        <div class="form-group popup-custom-width">
           <label class="inline-label" translate>Title</label>
+          <i popover-template="popup.titleTemplateUrl" 
+             popover-trigger="mouseenter" 
+             popover-placement="right"
+             class="fa fa-question-circle question-popover">
+          </i>
           <input type="text" class="form-control field-label" placeholder="Label" ng-model="block.title">
         </div>
 
-        <div class="form-group" ng-if="block.type === 'paragraphContent'">
```

Comment:
@OzzieOrca Note also this line gets dropped entirely (from master with merges):

`<textarea ng-model="block.content.paragraph" class="form-control" style="height:400px;"></textarea>`

The whole section was:
```
        <div class="form-group" ng-if="block.type === 'paragraphContent'">
          <textarea ng-model="block.content.paragraph" class="form-control" style="height:400px;"></textarea>
        </div>
```